cypress/e2e/ui/Modal-confirm-delete.cy.js

```diff
@@ -0,0 +1,32 @@
+import '@testing-library/cypress/add-commands';
+
+describe('MODAL CONFIRM DELETION', () => {
```

Comment:
Do you think we need to test the modals separately since the other tests are sort of already testing them by interacting with them to add/delete expenses like a user would?

Reply:
Strictly speaking, no we do not need it. But ....

The approach here with the `e2e/ui/` tests is based on the cypress docs best practices and also on the fact that splitting the test of the ui component itself from the logic flow of any app feature (e.g. add/delete expenses) allow us to make the test of the app feature faster and easier. 

See example from the modals, in add/delete we only click on the button we want whilst in the ui/modal test we also check that the component is correctly rendered (e.g. `it('checks that the modal is correctly rendered', ....)` ).

If you agree with this approach I think it would really speed up our app features tests

Reply:
Ah I see. Does it really make the feature test noticeable faster if you don't check the modal text at the same time? Also, checking that the month is correct in the modal seems like something the user would do when using the app, so maybe from that point of view it makes sense to also verify that it's correct when we test adding an expense.

Reply:
The way the Add Expense test is programmed, is basically also checking that the modal is correctly rendered, because otherwise the user (cypress) would not be able to click on `Open March 2022` if the month is `March 2023` for instance. 

Let me know if you prefer another approach. I'm open to create another issue to dig deeper on it if we want.